app/Main.hs

```diff
@@ -31,18 +28,29 @@ params = Params
           <> help "The testing directory where you're dumping your results.")
 
 versionOpt :: Parser (a->a)
-versionOpt = infoOption (showVersion version) 
+versionOpt = infoOption (showVersion version)
               (long "version"
-              <> short 'v'  
+              <> short 'v'
               <> help "Show version")
-            
 
---Update Files
+putStrColor :: Color -> String -> IO ()
+putStrColor color str = do
+  setSGR [SetColor Foreground Dull color]
+  putStr str
+  setSGR [Reset]
+
+putStrLnColor :: Color -> String -> IO ()
+putStrLnColor color str = do
+  setSGR [SetColor Foreground Dull color]
+  putStrLn str
+  setSGR [Reset]
+
+-- Update Files
 updateGolden :: FilePath -> IO ()
 updateGolden dir = do
-  putStrLn "Replacing golden with actual..."
+  putStrLnColor Yellow "Replacing golden with actual:"
```

Comment:
If I understand correctly, the color `Yellow` is used as a warning, and the color `Green` as success. I think we can get away with adding some alias functions like:
```haskell
success = putStrLnColor Green
warning = putStrLnColor Yellow
```
So, in the future, we know what is each color means.